lib/sidekiq/statistic/statistic/charts.rb

```diff
@@ -5,7 +5,7 @@ module Statistic
     class Charts < Base
       def information_for(type)
         worker_names.reverse.map.with_index do |worker, i|
-          color_hex = rgb_to_hex(color_for(worker))
+          color_hex = Helpers::Color.rgb_to_hex(Helpers::Color.color_for(worker))
```

Comment:
To avoid nested calls what do you think to a second parameter to the `for` method to explicitly tell it which result you want to receive?
```ruby
# The call
Helpers::Color.for(worker, format: :rgb)

# Helper::Color
def self.for(term, format: :plain); end
```